### PR TITLE
Update to use new log method in geoips

### PR DESCRIPTION
**Pull request by biosafetylvl5**
See https://github.com/NRLMMD-GEOIPS/geoips/pull/401 for more information. 

This PR removed manual logging with lots of `"********************"` string literals and replaces it with a built in logging function about to be implemented in geoips. This PR does not change any functionality, but some logged out statements will now have slightly different formatting.

Do not merge until the above pull request is merged into geoips main.

### CODE REVIEW

**Comment by biosafetylvl5** [comment]
^ Did I update the wrong release note file?

**Comment by evrose54** [comment]
@biosafetylvl5 I think you did update the wrong release note. You may need to update/create v1_12_2a0.rst release note for this one. If you look at https://github.com/NRLMMD-GEOIPS/geoips/blob/main/update_this_release_note, you'll be able to see the release note that should be update at the time of sending this message.

**Review by evrose54 — APPROVED** [review]
LGTM; we shouldn't merge this until we've merged this [PR](https://github.com/NRLMMD-GEOIPS/geoips/pull/401).

**Review by jsolbrig — APPROVED** [review]

**Pull request merged**
